### PR TITLE
chore: add check for vulnerabilities

### PR DESCRIPTION
**Pull request by frederic-huebner**

### CODE REVIEW

File: .github/workflows/cve-check.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+jobs:
+  trivy-vulnerability-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3
+
+      - name: Setup Gradle
+        uses: gradle/gradle-build-action@v2.5.0
```
Comment (christopher-cudennec) [inline .github/workflows/cve-check.yml:18]: Our other workflows use `setup-java`. What's the difference / benefit to use this action instead? Should we streamline the other workflows?

See https://github.com/SDA-SE/sda-dropwizard-commons/blob/master/.github/workflows/java-ci.yml#L29, for example.

**Review by christopher-cudennec — COMMENTED** [review]
* How can we ignore/accept found vulnerabilities?
* Nice to have: Post findings as comment to the PR.

**Comment by frederic-huebner** [comment]
> * How can we ignore/accept found vulnerabilities?
See here for example: https://github.com/SDA-SE/sda-dropwizard-commons/pull/2486/commits/c51c79eef84f29adc356e301d5608a847c063379#diff-b072ac81c042ceda99ba4a9e588dc9254ea9c37bed1727df018ed009ace33216
> * Nice to have: Post findings as comment to the PR.
I'll add it 👍

**Comment by frederic-huebner** [comment]
Post to the PR in case of findings:

Bildschirmfoto 2023-07-03 um 14 34 51


Commit status:

Bildschirmfoto 2023-07-03 um 14 35 15

**Review by JoergSiebahn — APPROVED** [review]

**Pull request merged**